test/utils/ReusableValues.scala

```diff
@@ -74,16 +74,37 @@ trait ReusableValues {
   )
 
   val underpaymentTypeRadioButtons = Seq(
-      createRadioButton("B00", UnderpaymentTypeMessages.importVAT),
-      createRadioButton("A00", UnderpaymentTypeMessages.customsDuty),
-      createRadioButton("E00", UnderpaymentTypeMessages.exciseDuty),
-      createRadioButton("A20", UnderpaymentTypeMessages.additionalDuty),
-      createRadioButton("A30", UnderpaymentTypeMessages.definitiveAntiDumpingDuty),
-      createRadioButton("A35", UnderpaymentTypeMessages.provisionalAntiDumpingDuty),
-      createRadioButton("A40", UnderpaymentTypeMessages.definitiveCountervailingDuty),
-      createRadioButton("A45", UnderpaymentTypeMessages.provisionalCountervailingDuty),
-      createRadioButton("A10", UnderpaymentTypeMessages.agriculturalDuty),
-      createRadioButton("D10", UnderpaymentTypeMessages.compensatoryDuty)
+    createRadioButton("B00", UnderpaymentTypeMessages.importVAT),
+    createRadioButton("A00", UnderpaymentTypeMessages.customsDuty),
+    createRadioButton("E00", UnderpaymentTypeMessages.exciseDuty),
+    createRadioButton("A20", UnderpaymentTypeMessages.additionalDuty),
+    createRadioButton("A30", UnderpaymentTypeMessages.definitiveAntiDumpingDuty),
+    createRadioButton("A35", UnderpaymentTypeMessages.provisionalAntiDumpingDuty),
+    createRadioButton("A40", UnderpaymentTypeMessages.definitiveCountervailingDuty),
+    createRadioButton("A45", UnderpaymentTypeMessages.provisionalCountervailingDuty),
+    createRadioButton("A10", UnderpaymentTypeMessages.agriculturalDuty),
+    createRadioButton("D10", UnderpaymentTypeMessages.compensatoryDuty)
+  )
+
+  val boxNumberRadioButtons = Seq(
```

Comment:
If `boxNumberRadioButtons` is only ever used in `BoxNumberViewSpec.scala`, it makes more sense to move it that class and not here because it's not reused anywhere.